pom.xml

```
@@ -19,6 +19,8 @@
   </description>
   <url>https://github.com/jenkinsci/cloudbees-folder-plugin/</url>
   <properties>
+    <revision>6.10.2</revision>
+    <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.130</jenkins.version>
```

Generally should bump to the next LTS (at least; at this point really 2.150.x is the oldest supportable baseline).
```suggestion
    <jenkins.version>2.150.3</jenkins.version>
```